BildCreatorLibrary/BildCreatorLibrary.csproj

```
@@ -0,0 +1,11 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>net5.0</TargetFramework>
```

```suggestion
    <TargetFramework>net6.0</TargetFramework>
```
Держите свои инструменты в порядке - в обновлённом современном состоянии.